packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx

```diff
@@ -35,6 +35,18 @@ export const KibanaPageTemplateInner: FC<Props> = ({
 }) => {
   let header;
 
+  const [offset, setOffset] = useState<number | undefined>(undefined);
+
+  useEffect(() => {
+    const kibanaChrome = document.querySelector('[data-test-subj="kibanaChrome"]') as HTMLElement;
```

Comment:
nit: I would make these selectors constants at the top.  I'm surprised they aren't already constants somewhere.